pynicotine/gtkgui/ui/settings/search.ui

```diff
@@ -365,62 +479,73 @@
                 <property name="visible">1</property>
-                <signal name="clicked" handler="on_clear_filter_history" swapped="no"/>
+                <property name="spacing">12</property>
                 <child>
-                  <object class="GtkBox">
+                  <object class="GtkButton" id="ClearFilterHistory">
                     <property name="visible">1</property>
-                    <property name="spacing">6</property>
+                    <signal name="clicked" handler="on_clear_filter_history" swapped="no"/>
                     <child>
-                      <object class="GtkImage">
+                      <object class="GtkBox">
                         <property name="visible">1</property>
-                        <property name="icon-name">edit-clear-symbolic</property>
-                      </object>
-                    </child>
-                    <child>
-                      <object class="GtkLabel">
-                        <property name="visible">1</property>
-                        <property name="label" translatable="yes">Clear Filter History</property>
-                        <property name="use-underline">1</property>
+                        <property name="spacing">6</property>
+                        <child>
+                          <object class="GtkImage">
+                            <property name="visible">1</property>
+                            <property name="icon-name">edit-clear-symbolic</property>
+                          </object>
+                        </child>
+                        <child>
+                          <object class="GtkLabel">
+                            <property name="visible">1</property>
+                            <property name="label" translatable="yes">Clear Filter History</property>
+                            <property name="tooltip_text" translatable="yes">Clears all default filters, and also all previous filters in the Results toolbar.</property>
+                            <property name="use-underline">1</property>
+                          </object>
+                        </child>
                       </object>
                     </child>
                   </object>
                 </child>
+                <child>
+                  <object class="GtkMenuButton" id="ShowSearchHelp">
+                    <property name="visible">1</property>
+                  </object>
+                </child>
               </object>
             </child>
+            <child>
+              <object class="GtkCheckButton" id="RegexpFilters">
+                <property name="label" translatable="yes">Allow advanced RegEx for Include and Exclude Text</property>
+                <property name="visible">1</property>
+                <property name="tooltip_text" translatable="yes">A RegEx (Regular Expression) is a sequence of meta-characters and characters, e.g. [ \d \D \s \S \w \W ] and operators { + * ? | ^ } ( )</property>
```

Comment:
A user thought that "regular" was the normal filter mode, so I have moved this option down to a lower position and given it a different label to disambiguate. Please review as I'm not sure how RegEx filtering works in N+